docs/howto/use-alternative-secrets-backend.rst

```diff
@@ -116,6 +116,28 @@ The following is a sample JSON file.
         }
     }
 
+The YAML file structure is similar to that of a JSON. The key-value pair of connection ID and the definitions of one or more connections.
+The connection can be defined as a URI (string) or JSON object.
+For a guide about defining a connection as a URI, see:: :ref:`generating_connection_uri`.
+For a description of the connection object parameters see :class:`~airflow.models.connection.Connection`.
+The following is a sample YAML file.
+
+.. code-block:: yaml
+
+    CONN_A: 'mysq://host_a'
```

Comment:
Can you use these examples in tests?  This will facilitate verification that everything is okay.